packages/create-guten-block/app/createGitignore.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Install a starter .gitignore
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const fs = require( 'fs-extra' );
+const shell = require( 'shelljs' );
+
+module.exports = (blockDir) => {
+	shell.cd( blockDir );
+	shell.touch( '.gitignore' );
```

Comment:
You sure the `.gitignore` file gets created in mac/win/linux here? Coz `.` at the start of the file mostly ended up in problem in my case.

Reply:
Didn't encounter any problems on Mac, Windows or Linux... See below for screenshots of the created `.gitignore` file. *Note: Mac one was taken after other PR updates, hence why output in file is different*

*Mac (v10.13)*
![fullscreen_9_25_18__1_28_pm](https://user-images.githubusercontent.com/7256402/46031520-0afa4980-c0c7-11e8-8476-9c744af39696.jpg)

*Windows (v10 Home*
![screenshot 1](https://user-images.githubusercontent.com/7256402/46030463-3b8cb400-c0c4-11e8-8aec-dff7e577a88e.png)

*Linux (Ubuntu 16.04 LTS)*
![screenshot from 2018-09-25 13-59-41](https://user-images.githubusercontent.com/7256402/46033204-97a70680-c0cb-11e8-98ef-f6cbd4bef3e9.png)

Reply:
Holly shit. Did you just check it on Mac/Windows/Linux? Best PR ever! 🙌